content/docs/guide.md

```diff
@@ -205,8 +204,6 @@ jobs:
       - main
 ```
 
-Packit user in Fedora is not a proven packager, so you need to grant [packit
-user](https://src.fedoraproject.org/user/packit) the ability to push.
-
-Creating pull requests is easy. `create_pr` defaults to true so the config
-starting this section is good enough.
+This option is ignored in Packit running on GitHub for security reasons and **is not**
+recommended either. Packit user in Fedora is not a proven packager, so you need
+to grant [packit user](https://src.fedoraproject.org/user/packit) the ability to push.
```

Comment:
If we are not covering  Bodhi updates, we don't need to request anything.
```suggestion
This option is ignored in Packit running on GitHub for security reasons and **is not**
recommended either.
```

Can you please add a final sentence that will nicely end this page?